main.tsp

```diff
@@ -15,11 +15,17 @@ enum Versions {
 }
 
 model UserSettings {
-  @doc("The computer username of the user.")
+  @doc("The real name of the user.")
   username: string;
+
+  @doc("The computer hostname of the user.")
```

Comment:
Hostname and username are not the same; hostname is the computer's name, and username is the user account. Worth double-checking if this field is meant to be the username.

Reply:
> A hostname is a label that is assigned to a device connected to a computer network and that is used to identify the device in various forms of electronic communication

https://en.wikipedia.org/wiki/Hostname